crates/vm/levm/src/vm.rs

```diff
@@ -371,19 +371,23 @@ impl VM {
 
     fn revert_create(&mut self) -> Result<(), VMError> {
         // Note: currently working with copies
-        let sender = self.call_frames.first().unwrap().msg_sender;
+        let sender = self
+            .call_frames
+            .first()
+            .ok_or(VMError::FatalUnwrap)?
+            .msg_sender;
         let mut sender_account = self.get_account(&sender);
 
         sender_account.info.nonce -= 1;
 
-        let new_contract_address = self.call_frames.first().unwrap().to;
+        let new_contract_address = self.call_frames.first().ok_or(VMError::FatalUnwrap)?.to;
 
         if self.cache.accounts.remove(&new_contract_address).is_none() {
             return Err(VMError::AddressDoesNotMatchAnAccount); // Should not be this error
         }
```

Comment:
We can abstract `self.call_frames.first().ok_or(VMError::FatalUnwrap)?` into a variable to avoid having it multiple times.

Reply:
https://github.com/lambdaclass/lambda_ethereum_rust/pull/1108/commits/c7a293ab826b8514b8142b41401a2e943a16cfc1